### PR TITLE
chore: taks-1 change text value

### PR DESCRIPTION
**Pull request by anonimak**
ini selesai task-1 untuk module task-1

### CODE REVIEW

File: task-1 [hunk 1]
```diff
@@ -1 +1 @@
-test task 1
+selesai buat task 1
```
Comment (clrssanbl) [inline task-1:1]: revisi

**Review by clrssanbl — CHANGES_REQUESTED** [review]

File: task-1 (hunk 1 above)
Comment (anonimak) [reply task-1:1]: oke

Comment (anonimak) [reply task-1]: done disini [c72695](https://github.com/anonimak/simulasi-pr/pull/1/commits/c72695d48cb81bbfb808132b40825cebd65428ce)

**Review by clrssanbl — APPROVED** [review]
oke

**Comment by anonimak** [comment]
mantul

**Pull request merged**